release/api/v1alpha1/bundle_types.go

```diff
@@ -147,31 +147,27 @@ type CertManagerBundle struct {
 type CoreClusterAPI struct {
 	Version    string   `json:"version"`
 	Controller Image    `json:"controller"`
-	KubeProxy  Image    `json:"kubeProxy"`
```

Comment:
This is a breaking change. I would just leave all the fields as they are, add a `ommitempty` tag and mark as deprecated with a doc comment

Reply:
Makes sense. Can we just retain the API struct for backward compatibility but remove its references from everywhere else in the codebase?